### DISCUSSION TITLE
[Feature] Creating tags on the fly

### DISCUSSION THREAD

**Discussion by SwallowYourDreams** [immich-app/immich#12179] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

First off, my thanks to the devs for finally implementing this useful and much sought-after feature! However, the way it is implemented now, you must first create a tag in a separate panel, using several more clicks, and only then can you search for and apply tags to images. I feel it would be much more useful to be able to create tags on the fly, meaning: skipping the extra clicks and creating new tags directly from the search input field.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by maxshcherbina** [comment]
I suspect this is probably already coming, but it's a good thing to mention!

**Comment by gregstah** [comment]
It's here and I just discovered it by mistake! 

In "info" "Add Tag" write your tag and "Enter" 😊
